index.html

```diff
@@ -0,0 +1,208 @@
+  </div>
+
+  <div class="sample">
+    <div id="demo-container" class="demo">
+      <example-snippet stamp-to="demo-container" highlight-as="html">
+          <template>
+              <model-viewer
+                src="assets/Astronaut.glb"
+                alt="A 3D model of an astronaut"
+                controls
+                auto-rotate
+                background-color="#455A64"></model-viewer>
+          </template>
+
+          <script>
+  console.log('This is how you log things to the console!');
+</script>
+
+      </example-snippet>
+    </div>
+
+    <div class="content">
+      <div class="wrapper">
+        <div class="center heading underline">
+          <h1 style="white-space:nowrap;"><span>&lt;</span>model-viewer<span>&gt;</span></h1>
+          <h3>Easily display interactive 3D models on the web & in AR</h3>
+        </div>
+
+        <h3 class="grouping-title">EXAMPLES</h3>
+        <ul id="list-example">
+          <li>
+            <a href="examples/default.html"><h4>Default</h4></a>
+            <p>The most basic configuration of model-viewer.</p>
+          </li>
+          <li>
+            <a href="examples/background-color.html"><h4>Background Color</h4></a>
+            <p>Using the background-color attribute to style the background color.</p>
+          </li>
+          <li>
+            <a href="examples/background-image.html"><h4>Background Image</h4></a>
+            <p>Using the background-image attribute to specify an equirectangular projection image.</p>
+          </li>
+          <li>
+            <a href="examples/augmented-reality.html"><h4>Augmented Reality</h4></a>
+            <p>Tests using configurations that enable viewing the model in augmented reality.</p>
+          </li>
+          <li>
+            <a href="examples/loading.html"><h4>Lazy Loading</h4></a>
+            <p>Examples using the different configurations of displaying an image while loading, and lazy loading models.</p>
+          </li>
+          <li>
+            <a href="examples/sources.html"><h4>Model Formats</h4></a>
+            <p>Tests different configurations of loading model formats.</p>
+          </li>
+          <li>
+            <a href="examples/multiple-elements.html"><h4>Multiple Elements</h4></a>
+            <p>Stress test of rendering 16 elements that require rendering on every frame, all in the view at the same time.</p>
+          </li>
+          <li>
+            <a href="examples/list.html"><h4>Rendering Test</h4></a>
+            <p>Test of many elements on a page, and lazily rendering only when in view.</p>
+          </li>
+          <li>
+            <a href="examples/pbr.html"><h4>Physically Based Rendering </h4></a>
+            <p>Showcase of high-resolution models with multiple textures and environment maps.</p>
+          </li>
+        </ul>
+
+        <h3 class="grouping-title">ATTRIBUTES</h3>
+
+        <ul class="list-attribute">
```

Comment:
I like the inclusion of the attributes here. Should we remove the list from `README.md` to avoid duplication?

Reply:
Sounds good to me. @cdata what do you think?

Reply:
Sounds fine as long as we link folks to the docs from the README

Reply:
Agreed. We need to take a large pass through the docs as part of #259, I've added a note there to check this as well.